charts/terraform-pod/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.7
+version: 0.0.7-bump
```

Comment:
You have no changes in chart, why have you bumped version?

Reply:
Unfortunately, helm chat releaser action tries to republish `terraform-pod` and `tekton-apps` charts, despite they have not changes. And if version exists (like terraform-pod:0.0.7), then it fails - does not publish chart, which I need. This is a workaround, and it can be ignored.